ipcontrol_connector.py

```diff
@@ -147,7 +147,7 @@ def _get_auth_token(self, action_result):
 
         ret_val, response = self._make_rest_call(auth_endpoint, action_result, method="post", data=data)
 
-        if phantom.is_fail(ret_val):
+        if (phantom.is_fail(ret_val)):
```

Comment:
why did we add extra parentheses to all of these `if` statements? did that happen because of a commit revert?

Reply:
yes, all the changes reverted and parentheses are available in the initial submitted code. This code will be released with py2 not updating the changes.